samples/net/rpl_border_router/README.rst

```diff
@@ -0,0 +1,136 @@
+
+	$ make pristine
+	$ make BOARD=<your-board>
+
+By default, the RPL border router application enables net shell support and
+provides some useful commands for debugging and viewing the network status.
+
+.. code-block:: console
+
+	shell> br repair
+	[rpl-br/shell] [INF] br_repair: Starting global repair...
+
+The **br repair** command will cause the RPL network to re-configure itself.
+
+.. code-block:: console
+
+	shell> net rpl
+	RPL Configuration
+	=================
+	RPL mode                     : mesh
+	Used objective function      : MRHOF
+	Used routing metric          : none
+	Mode of operation (MOP)      : Storing, no mcast (MOP2)
+	Send probes to nodes         : disabled
+	Max instances                : 1
+	Max DAG / instance           : 2
+	Min hop rank increment       : 256
+	Initial link metric          : 2
+	RPL preference value         : 0
+	DAG grounded by default      : no
+	Default instance id          : 30 (0x1e)
+	Insert Hop-by-hop option     : yes
+	Specify DAG when sending DAO : yes
+	DIO min interval             : 12 (4096 ms)
+	DIO doublings interval       : 8
+	DIO redundancy value         : 10
+	DAO sending timer value      : 4 sec
+	DAO max retransmissions      : 4
+	Node expecting DAO ack       : yes
+	Send DIS periodically        : yes
+	DIS interval                 : 60 sec
+	Default route lifetime unit  : 65535 sec
+	Default route lifetime       : 255
+
+	Runtime status
+	==============
+	Default instance (id 30) : 0xa80081e0 (active)
+	Instance DAGs   :
+	[ 1]* 2001:db8::1 prefix 2001:db8::/64 rank 256/65535 ver 255 flags GJ parent 0x00000000
+
+	No parents found.
+
+The **net rpl** command first prints out static compile time configururation
+settings. Then it prints information about runtime configuration of the system.
+
+.. code-block:: console
+
+	shell> net nbr
+	     Neighbor   Flags   Interface  State        Remain  Link                    Address
+	[ 1] 0xa80065e0 1/0/1/0 0xa8007140 reachable      2920  00:12:4B:00:00:00:00:01 fe80::212:4b00:0:1
+	[ 2] 0xa8006660 1/0/1/0 0xa8007140 stale             0  00:12:4B:00:00:00:00:03 fe80::212:4b00:0:3
+
+The **net nbr** command prints information about currently found IPv6 neighbor
+nodes. In this example there are two leaf nodes that are part of this RPL
+network.
+
+.. code-block:: console
+
+	shell> net route
+	IPv6 routes for interface 0xa8007140
+	====================================
+	IPv6 prefix : 2001:db8::212:4b00:0:3/128
+	        neighbor  : 0xa80065e0
+	        link addr : 00:12:4B:00:00:00:00:03
+	IPv6 prefix : 2001:db8::212:4b00:0:1/128
+	        neighbor  : 0xa8006660
+	        link addr : 00:12:4B:00:00:00:00:01
+
+The **nbr route** command prints information about currently found IPv6 routes.
+In this example all the nodes are directly connected to this RPL border router
+root node.
+
+If IEEE 802.15.4 support is enabled, then the **ieee15_4** command can be used
+to change the IEEE 802.15.4 network parameters like used channel or PAN id, if
```

Comment:
s/like/such as/

"like" is used for comparison ("run like the wind"), and "such as" is use for examples